x-pack/legacy/plugins/apm/server/lib/errors/get_trace_errors_per_transaction.ts

```diff
@@ -45,12 +45,14 @@ export async function getTraceErrorsPerTransaction(
       aggs: {
         transactions: {
           terms: {
-            field: TRANSACTION_ID
+            field: TRANSACTION_ID,
+            // high cardinality
+            execution_hint: 'map'
           }
         }
       }
     }
-  };
+  } as const;
```

Comment:
Why was this not needed before?

Reply:
TS widens 'map' to the string type, and then complains that it doesn't match the terms input type. `as const` prevents the widening.

Reply:
Makes sense